dev/AppLifecycle/AppInstance.cpp

```diff
@@ -578,19 +579,31 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
         // processes from seeing the mutex but not seeing the Key on the instance yet.
         auto releaseOnExit = m_dataMutex.acquire();
 
-        // We keep the mutex as a live member to ensure all other instances continue
+        // First caller creates the named mutex. All other instances continue
         // to get an 'open' instead of a 'create' due to it already existing.
-        // try_create returns true for a named mutex that already exists
-        // and so we can't rely on try_create and have to explicitly check GetLastError().
-        m_keyCreationMutex.create(mutexName.c_str(), 0, MUTEX_ALL_ACCESS);
+        wil::unique_mutex keyCreationMutex;
+        keyCreationMutex.create(mutexName.c_str(), 0, MUTEX_ALL_ACCESS);
 
-        bool currentIsKeyOwner = (GetLastError() != ERROR_ALREADY_EXISTS);
-        if (currentIsKeyOwner)
+        DWORD waitResult = 0;
+        // Wait for 0 seconds. If the mutex is taken we don't own the mutex
+        auto lock = keyCreationMutex.acquire(&waitResult, 0);
```

Comment:
Ok, so if the process crashes while owning the mutex, the ownership is reset.  Anybody waiting on the mutex gets a "wait abandoned" message. So is there some super slim possibility that some attempts to acquire will get "wait abandoned" meaning that the other person owning the mutex crashed, and it's OK for you to take ownership?